docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.timeline-item.scss

```diff
@@ -23,6 +23,33 @@
   }
 }
 
+// This adds the connected lines and dots to the Timeline Item within the Views pattern usage.
```

Comment:
Instead of duplicating these styles we should modify line 2 to include these classes:
```
.ptype-hs-timeline-item,
.views-view-pattern  .hb-timeline-item { … }
```

Reply:
Good call out, I think I meant to refactor this after I got it working but forgot. Thank you!